install-dependencies.sh

```diff
@@ -346,3 +362,24 @@ elif [ "$ID" == "arch" ]; then
     fi
     echo -e "Configure example:\n\t./configure.py\n\tninja release"
 fi
+
+CURL_ARGS=""
+if [ -x "$MINIO_BINARIES_DIR/minio" ]; then
+    echo "Minio server already downloaded"
```

Comment:
We should always download, so that running ./install-dependencies.sh leaves you with a version that is at least as new as the one when ./install-dependencies was updated. It doesn't affect the frozen toolchain, but it does affect users.